src/Sylius/Bundle/ResourceBundle/Behat/DefaultContext.php

```diff
@@ -340,4 +321,109 @@ protected function fixStepArgument($argument)
     {
         return str_replace('\\"', '"', $argument);
     }
+
+    /**
+     * @param NodeElement $table
+     * @param string $columnName
+     *
+     * @return integer
+     *
+     * @throws \Exception If column was not found
+     */
+    protected function getColumnIndex(NodeElement $table, $columnName)
+    {
+        $rows = $table->findAll('css', 'tr');
+
+        if (!isset($rows[0])) {
+            throw new \Exception("There are no rows!");
+        }
+
+        /** @var NodeElement $firstRow */
+        $firstRow = $rows[0];
+        $columns = $firstRow->findAll('css', 'th,td');
+        foreach ($columns as $index => $column) {
+            /** @var NodeElement $column */
+            if (0 === stripos($column->getText(), $columnName)) {
+                return $index;
+            }
+        }
+
+        throw new \Exception(sprintf('Column with name "%s" not found!', $columnName));
+    }
+
+    /**
+     * @param NodeElement $table
+     * @param array $fields
+     *
+     * @return NodeElement|null
+     *
+     * @throws \Exception If column was not found
+     */
+    protected function getRowWithFields(NodeElement $table, array $fields)
+    {
+        $foundRows = $this->getRowsWithFields($table, $fields, true);
```

Comment:
The method returns array of rows. According to docblock we should return ``` NodeElement ``` instead we return this array.